docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -12,17 +12,33 @@ and bronchiolitis, is a major global killer of both children and adults. Symptom
 include shortness of breath, weakness, fever, coughing and fatigue. It is important to check for a fever. Symptoms can last about 7 days and the infection is contagious
 to others shortly before and while experiencing symptoms. It is mainly
 caused by four microorganisms - Streptococcus pneumoniae (*pneumococcal
-pneumonia*), Haemophilus influenzae type B (*Hib*),influenza, and respiratory
+pneumonia*), Haemophilus influenzae type B (*Hib*), influenza, and respiratory
 syncytial virus (*RCV*). LRI can be caused by multiple pathogens and the pathogens may co-infect.
 Pneumococcal pneumonia is the largest cause of LRI
 mortality. [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
 
+Transmission of LRI may occur via several pathways, including direct physical contact,
+fomites, direct droplet spread, and suspended small particles. Intermingling of
+large numbers of people can facilitate transmission of respiratory pathogens. [CDC]_
+
+In GBD 2016, malnutrition was identified as a leading risk factor for lower respiratory infection
+mortality among children younger than 5 years and, together with air pollution (both household and ambient)
+and increased antibiotic use, was identified as a focus for targeted intervention measures. [Lancet]_
+
+.. todo::
+
+   Describe more about deaths and complications due to LRI.
+   Talk about current vaccination against influenza and pneumonia.
+   https://apps.who.int/iris/bitstream/handle/10665/241904/WER8714_129-144.PDFp
```

Comment:
Might also be nice to specify which organs comprise "lower respiratory tract". (Wiki says:
"
The lower respiratory tract or lower airway is derived from the developing foregut and consists of the trachea, bronchi (primary, secondary and tertiary), bronchioles (including terminal and respiratory), and lungs (including alveoli).[6] It also sometimes includes the larynx.
"
)

Reply:
Thanks for the suggestion! I will include it in this section.